### PR TITLE
Fix InterestGroupAuctionSupport

### PR DESCRIPTION
**Pull request by patmmccann**
intended to match ae as in use already

https://private-user-images.githubusercontent.com/1683175/301908471-e48bc335-ca6b-4702-a879-c49d9b26f316.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4OTE4MzUsIm5iZiI6MTcwNjg5MTUzNSwicGF0aCI6Ii8xNjgzMTc1LzMwMTkwODQ3MS1lNDhiYzMzNS1jYTZiLTQ3MDItYTg3OS1jNDlkOWIyNmYzMTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDJUMTYzMjE1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9M2I1NWM3M2FjYTQ1ZWM2NGE3NjA4ZmIxZjIzNWYyNjIzYjg5NzdlODFjNmYyNjU1NzQxNWIzNDkwMjA3YjMyOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.am2ON2huH_ItY01A6tj5N6M52JsFz8pAwIwQwTJqYys

### CODE REVIEW

File: 2.6.md [hunk 1]
```diff
@@ -3076,10 +3076,11 @@ This object allows sellers to signal interest group auction support for an Imp.
   </tr>
   <tr>
     <td><code>ae <b>PROVISIONAL</b></code></td>
-    <td>integer;<br>default 1</td>
+    <td>integer;<br>default 0</td>
     <td>Interest Group auction environment support for this impression: <br>
-1 = on-device orchestrated auction<br>
-2 = server-orchestrated auction<br>
+0 = server-side auction without interest groups<br>
```
Comment (dmdabbs) [inline 2.6.md:3081]: WIth the _InterestGroupAuctionSupport_ object there is no need for a "0" signal. When the _Imp_ presents this object there is some IG support. No object, no support.

Comment (patmmccann) [reply 2.6.md:3081]: Without this, you remove the ability for an exchange or publisher to say there is definitely not an interest group auction coming. Null suggests 'there might be one, who knows? '

Comment (dmdabbs) [reply 2.6.md:3081]: >Null suggests 'there might be one, who knows? '

By not sending an `InterestGroupAuctionSupport` object the exchange or publisher is signaling that there is no support. Should we add wording to that effect to make this explicit?

Comment (patmmccann) [reply 2.6.md:3081]: How would the SSP signal it doesn't know if there will be an interest group auction; which might be typical? This is currently how ae=1 is used by the prebid SSPs.

I am not sure I follow the objection to 0, in which one is explicitly signaling there will not be one. That is different than a missing object.

Comment (FlorentDancy) [reply 2.6.md:3081]: Agreed with @patmmccann here!

**Comment by FlorentDancy** [comment]
Fully support this change!

File: 2.6.md (hunk 1 above)
Comment (patmmccann) [reply 2.6.md:3081]: > Do you envision this to effectively signal there definitely not a runAdAuction call associated with that slot?

This seems useful to signal, no?

**Pull request closed without merging**